test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SpecializationE2ETests.cs

```diff
@@ -267,6 +269,21 @@ public async Task StartAsync_SetsCorrectActiveHost_RefreshesLanguageWorkerOption
             Assert.Equal("7", rpcChannelAfterSpecialization.Config.Description.DefaultRuntimeVersion);
         }
 
+        [Fact]
+        public void ColdStart_JitFailuresTest()
+        {
+            var path = Path.Combine(Path.GetDirectoryName(new Uri(typeof(HostWarmupMiddleware).Assembly.CodeBase).LocalPath), WarmUpConstants.PreJitFolderName, WarmUpConstants.JitTraceFileName);
+
+            var file = new FileInfo(path);
+
+            Assert.True(file.Exists, $"{file.FullName} does not exist!");
```

Comment:
nit: Can you please have a more descriptive error here? 
Something like: `$"Expected PGO file '{file.FullName} does not exist. The file was either renamed or deleted.`"